stacks/StorageStack.ts

```diff
@@ -1,25 +1,65 @@
-import { RDS, type StackContext } from "sst/constructs";
+import {
+  InstanceClass,
+  InstanceSize,
+  InstanceType,
+  SubnetType,
+  Vpc,
+} from "aws-cdk-lib/aws-ec2";
+import {
+  Credentials,
+  DatabaseInstance,
+  DatabaseInstanceEngine,
+  DatabaseSecret,
+  PostgresEngineVersion,
+} from "aws-cdk-lib/aws-rds";
+import { Duration } from "aws-cdk-lib/core";
+import type { StackContext } from "sst/constructs";
+
+const dbName = "birthdayBotDb";
+const dbUsername = "birthdayBotDbAdmin";
 
 export function StorageStack({ stack }: StackContext) {
   if (process.env.USE_LOCAL_DB) {
-    return {};
+    return null;
   }
 
-  const db = new RDS(stack, "Database", {
-    defaultDatabaseName: "birthdayBotDb",
-    engine: "postgresql13.9",
-    scaling: {
-      autoPause: stack.stage !== "production",
-    },
+  const vpc = new Vpc(stack, "vpc", {
+    subnetConfiguration: [
+      {
+        cidrMask: 24,
+        name: "public",
+        subnetType: SubnetType.PUBLIC,
+      },
+    ],
+  });
+
+  const dbSecret = new DatabaseSecret(stack, "RDS secret", {
+    username: dbUsername,
+    dbname: dbName,
   });
 
-  stack.addOutputs({
-    RDS_CLUSTER_ARN: db.clusterArn,
-    RDS_SECRET_ARN: db.secretArn,
-    RDS_DATABASE: db.defaultDatabaseName,
+  const db = new DatabaseInstance(stack, "Database", {
```

Comment:
We can use the sst provided RDS, but not sure that's make any difference with the current network issue. 
https://docs.sst.dev/constructs/RDS